docs/source/user-guide/gpu-support.md

```diff
@@ -7,7 +7,8 @@ available in Open Beta and is undergoing rapid development.
 ### System Requirements
 
 - NVIDIA Volta™ or higher GPU with [compute capability](https://developer.nvidia.com/cuda-gpus) 7.0+
-- CUDA 11 or CUDA 12
+- CUDA 12 (CUDA 11 support ends with RAPIDS v25.06; see
```

Comment:
Since these are docs, I think it's OK to tell users still on CUDA 11 that they can get CUDA 11 support if they're willing to stick to an older version of Polars/cudf-polars.

Reply:
It looks like you've got that below, maybe just mention that CUDA 11 users should see the note below.